### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by jursoleo**
[keywords.txt](https://github.com/arduino/library-registry/files/10613880/keywords.txt)
[Read.md](https://github.com/arduino/library-registry/files/10613881/Read.md)

### CODE REVIEW

**Review by per1234 — CHANGES_REQUESTED** [review]
Hello @jursoleo. Thanks for your interest in contributing to the Arduino Library Manager index.

It appears you may be attempting to submit a library, but this is not the right way to do that. Please carefully read the instructions here and then update your pull request to submit the library in the correct manner:

https://github.com/arduino/library-registry#adding-a-library-to-library-manager

Please let me know if you have any questions or problems while following the instructions.

If you decide you no longer wish to submit the library, you can close this pull request.

**Comment by jursoleo** [comment]
I did try to follow the instructions … exactly.  However when I was supposed to do a ‘compare …  ‘    as instructed,  there was no COMPARE button or link or menu item that I could find.  So it is hard to follow instructions when the reality does not match the document.

I will try one more time … before I give up.

 

Thanks

**Comment by jursoleo** [comment]
Well time to stop wasting time on this.  I have no idea what to do next, don’t know what issues are still pending,  and why my library does not appear in the Arduino IDE.  The tools and process are completely incomprehensible,  and when I try to follow the instructions,  after a step or two,  I am told to do something that has no obvious way to accomplish.   I don’t want to publish my library if it takes two days to upload it.  I will allow my users to download it from my site and manually install.

 

GitHub appears to be a big waste of time to me.  Unless you happen to be  the person who designed this convoluted mess .. there is no way to figure out how to operate it, how to identify the status of the process, and what it wants to proceed.

 

John Ursoleo/Versalent

**Comment by per1234** [comment]
Hello @jursoleo. I'm sorry you found the submission process unfriendly.

I would be interested in trying to improve the documentation. Your perspective as a new user of the library submission system and GitHub in general is valuable. My deep familiarity with both of them is actually a disadvantage when it comes to writing documentation that is clear to the reader with less familiarity. Unfortunately I am not sure which parts of the current documentation is unclear or incorrect. I just followed them step by step and everything in the instructions matched the required procedures on GitHub. So I would need some more detailed information from you if I am to take action in response to your feedback.

>when I was supposed to do a ‘compare …  ‘    as instructed,  there was no COMPARE button or link or menu item that I could find.  So it is hard to follow instructions when the reality does not match the document.

Please tell me which exact step in the instructions are you referring to? There is no mention of a "COMPARE button" in the instructions.

>when I try to follow the instructions,  after a step or two,  I am told to do something that has no obvious way to accomplish.

Please tell me exactly which parts of the instructions you encountered this with. What did you try? What did you observe when you tried it that did not match your expectations?

>I don’t want to publish my library if it takes two days to upload it.

Assuming the library itself is compliant with the requirements (which is important regardless of whether you are submitting it to Library Manager), the submission process would be expected to take less than five minutes. There are only six actions to perform. The first is clicking a link. Four out of five are clicking a button. The most complex is typing or pasting a URL into a text file.

>GitHub appears to be a big waste of time to me.

You are free to have that opinion, but millions of professional and hobby developers would disagree. The great majority of Arduino projects, and software projects in general, are hosted here on GitHub. So learning to submit a pull request on GitHub has value to a software developer beyond having their library added to Arduino Library Manager alone.
